service-config/src/main/java/com/palantir/remoting2/config/service/ProxyConfiguration.java

```diff
@@ -36,45 +36,105 @@
 @Style(visibility = Style.ImplementationVisibility.PACKAGE, builder = "new")
 public abstract class ProxyConfiguration {
 
+    enum Type {
+
+        /**
+         * Use a direct connection. This option will bypass any JVM-level configured proxy settings.
+         */
+        direct,
```

Comment:
These leak into the configuration, which usually doesn't contain upper case values.  Glad to change, but that's why I had made this particular enum lowercase